wiki/CHEATSHEET.md

```diff
@@ -84,6 +84,8 @@ What I want | How to get it
 --- | ---
 Remove nulls | `StreamEx/EntryStream.nonNull()`
 Remove entries which keys or values are null | `EntryStream.nonNullKeys()/nonNullValues()`
+Remove entries which keys are equal to any of the supplied keys | `EntryStream.withoutKeys()`
```

Comment:
I think it should be "whose keys/whose values" (yes, I used the wrong word before as well)